doc/03-Configuration.md

```diff
@@ -24,6 +24,7 @@ Configuration of the database used by Icinga DB.
 
 Option                   | Description
 -------------------------|-----------------------------------------------
+type                     | **Required.** Either "mysql" or "pgsql".
```

Comment:
Introducing `type` as required will break all installations. I would make it optional and leave it as `mysql` by default.